src/blosc2/ndarray.py

```diff
@@ -3766,6 +3766,49 @@ def matmul(x1: NDArray, x2: NDArray, **kwargs: Any) -> NDArray:
     return result.squeeze()
 
 
+def transpose(x, **kwargs: Any) -> NDArray:
+    """
+    Returns a Blosc2 NDArray with axes transposed.
+
+    Parameters
+    ----------
+    x: `NDArray`
+        The input array.
+    kwargs: Any, optional
+        Keyword arguments that are supported by the :func:`empty` constructor.
+
+    Returns
+    -------
+    out: :ref:`NDArray`
```

Comment:
Why are you using `:ref:NDArray` here and a plain `NDArray` above?  Look into other places for similar docstrings and try to be consistent.